### PR TITLE
Log app settings on startup

### LINKED ISSUE #914: Print settings information in the logs on startup
It's important to know the app version and configuration data of the app. Instead of asking for a screenshot or the settings screen from the user its easier to print them in the logs on startup.

related to https://github.com/get10101/10101/issues/912

### PR DESCRIPTION
**Pull request by luckysori**
Fixes #914.

### CODE REVIEW

File: mobile/lib/main.dart [hunk 1]
```diff
@@ -396,3 +399,26 @@ class ScaffoldWithNavBar extends StatelessWidget {
...
+    FLog.info(text: "Built on branch: $branch");
+  }
+
+  PackageInfo packageInfo = await PackageInfo.fromPlatform();
+  FLog.info(text: "Build number: ${packageInfo.buildNumber}");
+  FLog.info(text: "Build version: ${packageInfo.version}");
+
+  FLog.info(text: "Network: ${config.network}");
+  FLog.info(text: "Esplora endpoint: ${config.esploraEndpoint}");
+  FLog.info(text: "Coordinator: ${config.coordinatorPubkey}@${config.host}:${config.p2PPort}");
```
Comment (holzeis) [inline mobile/lib/main.dart:420]: 🔧 Can you add oracle endpoint and pubkey?

**Review by holzeis — APPROVED** [review]
Nice, thanks!

File: mobile/lib/main.dart (hunk 1 above)
Comment (luckysori) [reply mobile/lib/main.dart:420]: Good idea!

**Pull request merged**